pkg/sdn/api/types.go

```diff
@@ -60,3 +60,33 @@ type NetNamespaceList struct {
 	unversioned.ListMeta
 	Items []NetNamespace
 }
+
+// EgressFirewallPolicy gives the type of an EgressFirewallRule
+type EgressFirewallPolicy string
+
+const (
+	EgressFirewallPolicyAllow EgressFirewallPolicy = "allow"
+	EgressFirewallPolicyDeny  EgressFirewallPolicy = "deny"
+)
+
+// EgressFirewallRule contains a single outgoing traffic firewall rule
+type EgressFirewallRule struct {
+	Policy EgressFirewallPolicy
+	From   string // Namespace name or empty
```

Comment:
We don't do that for quota, which is the same outcome.  All users
should be using new-project (create project directly is admin), which
uses the template, so admins can't forget.